pkg/linter/linter_test.go

```diff
@@ -773,7 +773,7 @@ func BenchmarkRegalLintingItself(b *testing.B) {
 
 	var rep report.Report
 
-	for i := 0; i < b.N; i++ {
+	for range b.N {
```

Comment:
This doesn't work in Go 1.22 which is what we target.. but I guess since these are test files, it doesn't matter.

Reply:
Hmm, I wonder why golangcilint wasn't able to work that out? perhaps because we have a more recent toolchain version?

Reply:
Yeah, that sounds reasonable, and smart :) If it did this generally, we'd have a lot of places to update I reckon.